lib/api/unittest/CDataFrameAnalyzerTrainingTest.cc

```diff
@@ -38,6 +38,7 @@
 
 #include <rapidjson/prettywriter.h>
 
+#include <boost/make_shared.hpp>
```

Comment:
Why not `std::shared_ptr`?

Reply:
`CLogger::reconfigure` expects `boost::shared_ptr`. I wouldn't change the interfaces there so we won't unnecessarily diverge further from `main`.

Reply:
We have to keep using `boost::shared_ptr` for passing arguments to Boost::Log, because Boost::Log still uses them.

Reply:
Thanks!